Sources/Helpers/TranslationsHelper.swift

```diff
@@ -129,14 +143,20 @@ open class TranslationsHelper: NSObject {
         }
     
         isLoading = true
-        
-        Manager.content.sync(query: syncQuery) { moduleId, error in
-            self.processSyncResult(moduleId: moduleId, error: error)
-            self.completionHandlers.forEach { $0(error) }
+        if let id = moduleId {
+            Manager.content.sync(query: syncQuery) { moduleId, error in
+                self.processSyncResult(moduleId: id, error: error)
+                self.completionHandlers.forEach { $0(error) }
```

Comment:
To avoid duplicating this line, make an inline closure/block/lambda and call it here and on the other place where used